### PR TITLE
Input tabs

### PR DESCRIPTION
**Pull request by ianjennings**
Adds a tab switcher for inputs. There's a problem right now though when you switch tabs it doesn't alter state?

<a href="https://app.paircast.io/replay/69a9ce27-584b-42cc-ba89-6bca40fadcfe?shareKey=7OXUupseuU3YKg1Y3BeYlA#0:180000"></a>

tabs

### CODE REVIEW

**Review by wlawt — COMMENTED** [review]
Does this PR also intend on addressing the state issue mentioned in the description?

**Comment by ianjennings** [comment]
No it doesn't fix that issue

File: src/components/Output.js [hunk 1]
```diff
@@ -9,16 +11,21 @@ export function Output({
   updateBabelConfig,
   removeBabelConfig,
 }) {
-  return babelConfig.map((config, index) => {
-    return (
-      <CompiledOutput
+  let panes = babelConfig.map((config, index) => {
+
+    console.log(index)
```
Comment (wlawt) [inline src/components/Output.js:16]: We should remove before merge

File: src/components/Output.js [hunk 2]
```diff
@@ -9,16 +11,21 @@ export function Output({
...
+
+    return {
+      menuItem: 'Config ' + index, render: () => <CompiledOutput
         source={debouncedSource}
         customPlugin={enableCustomPlugin ? customPlugin : undefined}
         config={config}
         key={index}
         onConfigChange={config => updateBabelConfig(config, index)}
         removeConfig={() => removeBabelConfig(index)}
       />
```
Comment (wlawt) [inline src/components/Output.js:26]: Do we want the `Config` tab width to take up the entire width, or leave with the current (taking up about half)?

image

Comment (ianjennings) [reply src/components/Output.js:26]: Lol, yeah it should be 100% I think.

**Comment by ianjennings** [comment]
Closed by #98

**Pull request closed without merging**